File: Moja stacja pogodowa/Models/Widget/WidgetModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Moja_stacja_pogodowa.Models.Widget
{
    public class WidgetModel
    {
        public int Id { get; set; }
        public string UserId { get; set; }
        public int APIId { get; set; }
        public string Name { get; set; }
        public string Lat { get; set; }
        public string Long { get; set; }
        public string CityId { get; set; }
    }
}

File: moja-stacja-pogodowa-web/src/app/models/widget.model.ts
import { CityOpenWeather } from './city-open-weather/city-open-weather.model';

export class Widget {
  APIId: number;
  name: string;
  lat: string;
  long: string;
  city: CityOpenWeather;
}

export class WidgetApi {
  id: number;
  userId: string;
  APIId: number;
  name: string;
  lat: string;
  long: string;
  cityId: number;
}
export class WidgetApiModel {
  userId: string;
  APIId: number;
  name: string;
  lat: string;
  long: string;
  cityId: string;
}
